Formula/rocketmq.rb

```diff
@@ -0,0 +1,133 @@
+      s.gsub! "export CLASSPATH=.:${BASE_DIR}/conf:${CLASSPATH}",
+        "export CLASSPATH=.:${BASE_DIR}/lib/*:${BASE_DIR}/conf:${CLASSPATH}"
+      s.gsub! "JAVA_OPT=\"${JAVA_OPT} -Djava.ext.dirs", "#JAVA_OPT=\"${JAVA_OPT} -Djava.ext.dirs"
+      s.gsub! "choose_gc_log_directory\n", ""
+      s.gsub! "choose_gc_options\n", "\n"
+    end
+
+    inreplace "bin/runbroker.sh" do |s|
+      s.gsub! "export CLASSPATH=.:${BASE_DIR}/conf:${CLASSPATH}",
+        "export CLASSPATH=.:${BASE_DIR}/lib/*:${BASE_DIR}/conf:${CLASSPATH}"
+      s.gsub! "JAVA_OPT=\"${JAVA_OPT} -Djava.ext.dirs", "#JAVA_OPT=\"${JAVA_OPT} -Djava.ext.dirs"
+      s.gsub! "JAVA_OPT=\"${JAVA_OPT} -verbose:gc -Xloggc:${GC_LOG_DIR}/rmq_broker_gc_%p_%t.log -XX:+PrintGCDetails" \
+        " -XX:+PrintGCDateStamps -XX:+PrintGCApplicationStoppedTime -XX:+PrintAdaptiveSizePolicy\"", ""
+      s.gsub! "JAVA_OPT=\"${JAVA_OPT} -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=5 -XX:GCLogFileSize=30m\"", ""
+      s.gsub! "JAVA_OPT=\"${JAVA_OPT} -XX:-UseLargePages -XX:-UseBiasedLocking\"",
+        "JAVA_OPT=\"${JAVA_OPT} -XX:-UseLargePages\""
+      s.gsub! "choose_gc_log_directory\n", "\n"
+    end
+
+    inreplace "bin/tools.sh" do |s|
+      s.gsub! "export CLASSPATH=.:${BASE_DIR}/conf:${CLASSPATH}",
+        "export CLASSPATH=.:${BASE_DIR}/lib/*:${BASE_DIR}/conf:${CLASSPATH}"
+      s.gsub! "JAVA_OPT=\"${JAVA_OPT} -Djava.ext.dirs", "#JAVA_OPT=\"${JAVA_OPT} -Djava.ext.dirs"
+    end
+
+    # netty-all-4.0.42.Final.jar doesn't work on arm64 macOS
+    rm_rf "lib/netty-all-4.0.42.Final.jar"
+    libexec.install "lib"
+
+    # install netty-all-4.1.65.Final.jar which works on arm64 macOS
+    resource("netty-all").stage { cp "netty-all-4.1.65.Final.jar", libexec/"lib" }
+
+    # remove non-executable
+    rm_rf "bin/dledger"
+    # remove Windows scripts
+    rm_r Dir.glob("bin/*.cmd")
+
+    prefix.install "bin"
+    bin.env_script_all_files(libexec/"bin", Language::Java.java_home_env)
+
+    inreplace "conf/logback_namesrv.xml" do |s|
+      s.gsub! %r{\$\{user\.home\}/logs/rocketmqlogs}, "#{var}/log/rocketmq"
+    end
+
+    inreplace "conf/logback_broker.xml" do |s|
+      s.gsub! %r{\$\{user\.home\}/logs/rocketmqlogs}, "#{var}/log/rocketmq"
+    end
+
+    mv "conf", "rocketmq"
+    etc.install "rocketmq"
+    libexec.install_symlink(etc/"rocketmq" => "conf")
+
+    rm etc/"rocketmq/broker.conf"
+    (etc/"rocketmq/broker.conf").write broker_conf
+  end
+
+  def post_install
+    (var/"rocketmq").mkpath
+    (var/"log/rocketmq").mkpath
+    (var/"rocketmq/commitlog").mkpath
+  end
+
+  def broker_conf
+    <<~EOS
+      brokerClusterName = DefaultCluster
+      brokerName = broker-a
+      brokerId = 0
+      deleteWhen = 04
+      fileReservedTime = 48
+      brokerRole = ASYNC_MASTER
+      flushDiskType = ASYNC_FLUSH
+      storePathRootDir=#{var}/rocketmq
+      storePathCommitLog=#{var}/rocketmq/commitlog
+      autoCreateTopicEnable=true
+    EOS
+  end
```

Comment:
We don't want to ship configuration if upstream doesn't

Reply:
Upstream does ship configuration, added `storePathRootDir`, `storePathCommitLog` path to fit Homebrew.